.dotnet/src/Generated/Models/CreateChatCompletionRequest.Serialization.cs

```diff
@@ -1,19 +1,15 @@
 // <auto-generated/>
 
-#nullable disable
```

Comment:
We don't need `#nullable disable` if `// <auto-generated/>` is used, per https://learn.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies.  It doesn't hurt, but when we later turn on nullable annotations, we'll either need to remove `// <auto-generated/>` or add `#nullable enable`.